src/renderer/components/NodeDocumentation/docsMarkdown.tsx

```diff
@@ -12,7 +12,7 @@ const getDocsMarkdownComponents = (interactive: boolean): Components => {
         p: ({ children }) => {
             return (
                 <Text
-                    my={2}
+                    // my={2}
```

Comment:
Why?

Reply:
I put everything in a VStack that has this margin instead.

Reply:
That's the issue. You can't remove the margin here, because that is necessary for markdown formatting. You just removed all spacing between paragraphs. 

It's cool that your VStack has margin, but now nothing else has :)

Reply:
Oooooh oops